### PR TITLE
Added missing punctuation's in documentation

### PR DESCRIPTION
**Pull request by sumitkharche**
#### Added missing punctuation's into the below sections of documentation:

- Getting-Started
- Extending
- Implementation
- Reference

### CODE REVIEW

File: Getting-Started/Code/Creating-Forms/index.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ namespace MyFirstForm.Controllers
 }
 ```
 
-If the model state is invalid, `CurrentUmbracoPage()` will send the user back to the form. If valid, you can work with the form data (e.g. sending an email to site admin) and then `RedirectToCurrentUmbracoPage();`
+If the model state is invalid, `CurrentUmbracoPage()` will send the user back to the form. If valid, you can work with the form data. (e.g. sending an email to site admin) and then `RedirectToCurrentUmbracoPage();`
```
Comment (sofietoft) [inline Getting-Started/Code/Creating-Forms/index.md]:
```suggestion
If the model state is invalid, `CurrentUmbracoPage()` will send the user back to the form. If valid, you can work with the form data, e.g. sending an email to site admin and then `RedirectToCurrentUmbracoPage();`.
```

File: Getting-Started/Design/Rendering-Content/index.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ You can do this by querying content relative to your current page in template vi
 </ul>
 ```
 
-You can use the Query Builder in the template editor to build more advanced queries
+You can use the Query Builder in the template editor to build more advanced queries:
```
Comment (sofietoft) [inline Getting-Started/Design/Rendering-Content/index.md]:
```suggestion
You can use the Query Builder in the template editor to build more advanced queries.
```

File: Getting-Started/Design/Templates/index-v7.md [hunk 3]
```diff
@@ -9,13 +9,13 @@ _Templating in Umbraco builds on the concept of Razor Views from ASP.NET MVC - i
 ## Creating your first template
 By default all document types should have a template attached - but in case you need an alternative template or a new one, you can create one:
 
-Open the settings section inside the Umbraco backoffice and right-click the **templates** folder. then choose **create**. Enter a template name and click the create button. You will now see the default template markup in the backoffice template editor
+Open the settings section inside the Umbraco backoffice and right-click the **templates** folder. then choose **create**. Enter a template name and click the create button. You will now see the default template markup in the backoffice template editor:
```
Comment (sofietoft) [inline Getting-Started/Design/Templates/index-v7.md]:
```suggestion
Open the settings section inside the Umbraco backoffice and right-click the **templates** folder. then choose **create**. Enter a template name and click the create button. You will now see the default template markup in the backoffice template editor.
```

File: Getting-Started/Design/Templates/index-v7.md [hunk 4]
```diff
@@ -9,13 +9,13 @@ _Templating in Umbraco builds on the concept of Razor Views from ASP.NET MVC - i
...
 
 ![Created template](images/create-template.png)
 
 
 ## Allowing a template on a document type
-To use a template on a document, you must first allow it on the content's type. Open the document type you want to use the template and check the template under "allowed templates"
+To use a template on a document, you must first allow it on the content's type. Open the document type you want to use the template and check the template under "allowed templates":
```
Comment (sofietoft) [inline Getting-Started/Design/Templates/index-v7.md]:
```suggestion
To use a template on a document, you must first allow it on the content's type. Open the document type you want to use the template and check the template under "allowed templates".
```

File: Getting-Started/Design/Templates/index.md [hunk 5]
```diff
@@ -9,13 +9,13 @@ _Templating in Umbraco builds on the concept of Razor Views from ASP.NET MVC - i
 ## Creating your first template
 By default all document types should have a template attached - but in case you need an alternative template or a new one, you can create one:
 
-Open the settings section inside the Umbraco backoffice and right-click the **templates** folder. then choose **create**. Enter a template name and click the create button. You will now see the default template markup in the backoffice template editor
+Open the settings section inside the Umbraco backoffice and right-click the **templates** folder. then choose **create**. Enter a template name and click the create button. You will now see the default template markup in the backoffice template editor:
```
Comment (sofietoft) [inline Getting-Started/Design/Templates/index.md]:
```suggestion
Open the settings section inside the Umbraco backoffice and right-click the **templates** folder. then choose **create**. Enter a template name and click the create button. You will now see the default template markup in the backoffice template editor.
```

File: Getting-Started/Design/Templates/index.md [hunk 6]
```diff
@@ -9,13 +9,13 @@ _Templating in Umbraco builds on the concept of Razor Views from ASP.NET MVC - i
...
 
 ![Created template](images/create-template-v8.png)
 
 
 ## Allowing a template on a document type
-To use a template on a document, you must first allow it on the content's type. Open the document type you want to use the template and select the template under "allowed templates"
+To use a template on a document, you must first allow it on the content's type. Open the document type you want to use the template and select the template under "allowed templates":
```
Comment (sofietoft) [inline Getting-Started/Design/Templates/index.md]:
```suggestion
To use a template on a document, you must first allow it on the content's type. Open the document type you want to use the template and select the template under "allowed templates".
```

File: Getting-Started/Code/Creating-Forms/index.md (hunk 1 above)
Comment (sumitkharche) [reply Getting-Started/Code/Creating-Forms/index.md]: That is looking good to me. I will go ahead and commit the changes.

File: Getting-Started/Design/Rendering-Content/index.md (hunk 2 above)
Comment (sumitkharche) [reply Getting-Started/Design/Rendering-Content/index.md]: That is looking good to me. I will go ahead and commit changes.

File: Getting-Started/Design/Templates/index-v7.md (hunk 4 above)
Comment (sumitkharche) [reply Getting-Started/Design/Templates/index-v7.md]: Changes looks good to me. I will go ahead and commit changes.

File: Getting-Started/Design/Templates/index.md (hunk 5 above)
Comment (sumitkharche) [reply Getting-Started/Design/Templates/index.md]: Changes looks good to me. I will go ahead and commit changes.

File: Getting-Started/Design/Templates/index.md (hunk 6 above)
Comment (sumitkharche) [reply Getting-Started/Design/Templates/index.md]: Changes looks good to me. I will go ahead and commit changes.

File: Getting-Started/Design/Templates/index-v7.md (hunk 3 above)
Comment (sumitkharche) [reply Getting-Started/Design/Templates/index-v7.md]: Changes looks good to me. I will go ahead and commit changes.

**Comment by sumitkharche** [comment]
@sofietoft Thanks for the quick review. 
The changes you have suggested are looking good to me. I have also committed the changes.

**Comment by sofietoft** [comment]
Thanks! 😄 🎉 

Unfortunately it doesn't meet the [quality standard guidelines for Umbraco Hacktoberfest swag](https://umbraco.com/blog/hacktoberfest-2019/).

We look forward to seeing more contributions from you 💪

**Pull request merged**